pkg/projects/Microsoft.NETCore.App/netcoreapp1.1/project.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
per our in person discussion, I removed the netcoreapp1.1 specific stuff and it looks like it still works. The main thing I needed to change for netcoreapp1.1 to work was that I needed up update the version of NetStandardLibrary being used.